### PR TITLE
Added News Data Provider module.

### PR DESCRIPTION
**Pull request by heymonicakay**
### Changes Made

1. Added file `news/NewsProvider.js`

### Steps to Review

```
git fetch --all
git checkout mp-news-provider
```
- [ ] From Terminal > Navigate to `api` folder in repo > run `json-server -p 8088 -w database.json`
- [ ] Open new Terminal window >  Navigate to `src` folder in repo > Run `serve` > (⌘) + 'double-click' on Local URL.
- [ ] Test app functionality - no changes expected.
- [ ] View code > Confirm file modifications.

### CODE REVIEW

File: src/scripts/news/NewsProvider.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+let articles = []
+
+const eventHub = document.querySelector(".main")
```
Comment (mswinkels09) [inline src/scripts/news/NewsProvider.js]: change the query selector to target ".container" to match index.html

File: src/scripts/news/NewsProvider.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+let articles = []
```
Comment (mgjeffries) [inline src/scripts/news/NewsProvider.js]: I like the calling these objects articles. Should we rename this file to ArticleProvider?

File: src/scripts/news/NewsProvider.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+const eventHub = document.querySelector(".main")
+
+const dispatchNewsStateChangeEvent = () => {
```
Comment (mgjeffries) [inline src/scripts/news/NewsProvider.js]: ... we could change this to articleChangeEvent

File: src/scripts/news/NewsProvider.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        }
+      )
+}
+
+export const useNews = () => {
+  return articles.slice()
+}
+
+export const getNews = () => {
+  return fetch('http://localhost:8088/articles')
```
Comment (mgjeffries) [inline src/scripts/news/NewsProvider.js]: And we could change these functions use the term "articles."

**Review by mswinkels09 — APPROVED** [review]

**Review by mgjeffries — APPROVED** [review]

**Pull request merged**